lib/src/cli/deployapps/deleteapps_command.dart

```diff
@@ -0,0 +1,84 @@
+////////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2022 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:args/command_runner.dart';
+
+import 'options.dart';
+import 'baas_client.dart';
+
+class DeleteAppsCommand extends Command<void> {
+  @override
+  final String description = 'Delete test applications from MongoDB Atlas.';
+
+  @override
+  final String name = 'delete-apps';
+
+  @override
+  bool get hidden => true;
+  
+  late Options options;
+
+  DeleteAppsCommand() {
+    populateOptionsParser(argParser).addOption("appIds", help: "List of appIds of deployed apps on MongoDB Atlas.");
+  }
+
+  @override
+  FutureOr<void>? run() async {
+    options = parseOptionsResult(argResults!);
+
+    var result = argParser.parse(argResults!.arguments);
+    List<String> appIds = (result['appIds'] as String).split(',');
+
+    if (options.atlasCluster != null) {
+      if (options.apiKey == null) {
+        abort('--api-key must be supplied when --atlas-cluster is not set');
+      }
+
+      if (options.privateApiKey == null) {
+        abort('--private-api-key must be supplied when --atlas-cluster is not set');
+      }
+
+      if (options.projectId == null) {
+        abort('--project-id must be supplied when --atlas-cluster is not set');
+      }
+    }
+    if (appIds.isEmpty) {
+      abort('--appIds must be supplied');
+    }
+
+    final differentiator = options.differentiator ?? 'local';
```

Comment:
Why do we need to pass a differentiator here if we're passing the final app id to be deleted? The differentiator won't be used at all.

Reply:
I changed the implementation the apps to be discovered by differentiator instead of appIds. I did this in order to be able to move all the deployments before all the tests.